### ISSUE TITLE
Best model tracking operator

### ISSUE THREAD

**Issue by HeuristicLab-Trac-Bot** [heal-research/HeuristicLab#194]
*Issue migrated from trac ticket # 194*

**milestone:** HeuristicLab 3.3.0 | **component:** StructureIdentification | **priority:** medium | **resolution:** done

#### 2008-07-04 15:35:56: @gkronber created the issue

___
> Like in HL2 the best models for the training set and the validation set should be tracked over the whole GP run.
